### ISSUE TITLE
Insert [ ] in New Note Page

### ISSUE THREAD

**Issue by mseliger** [danny0838/firefox-scrapbook#200]
I've a problem with ScrapBook X, using Pale Moon 27.2.1 (but also in older versions) in combination with the windows operating system.
When I create a "New Note Page" I often want to use brackets, this ones: [ ]. But when I enter them in a note page they are always convert with {THIS} (I think it's from the HTML templates). This behaviour is only on a windows operating system, with Lubuntu the brackets are just inserted and not replaced.
Perhaps it is a key combination defined in windows-version of ScrapBook?

**Comment by danny0838** [comment]
You mean when you press `[`, `[` appears, and then you press ` `, and ` ` appears, and then you press `]`, and the previous `[ ` became `{this}`? If not could you describe the detail of how it happen, possibly a few screenshots illustrating it?

**Comment by mseliger** [comment]
Yes, I press the key "[" and it will change immediately in "{this}" and if I press "]" it will change immediately in "{this}". Strange.

**Comment by danny0838** [comment]
I cannot reproduce it and what you told seems to be a bit different with 2F. Could you provide a bit screenshots illustrating it. Additionally, could you confirm whether this issue happens on Firefox?
